utils/merge_syft_sbom.py

```diff
@@ -163,7 +163,7 @@ def merge_sboms(cachi2_sbom_path: str, syft_sbom_path: str) -> str:
     is_duplicate_component = _get_syft_component_filter(cachi2_sbom["components"])
 
     filtered_syft_components = [
-        component for component in syft_sbom["components"] if not is_duplicate_component(component)
+        component for component in syft_sbom.get("components", []) if not is_duplicate_component(component)
```

Comment:
**Commit message:**

Please include a reference to the CycloneDX spec when referring to it, I guess this is the one (since we can only work with 1.5 and 1.4 and default to 1.4 if I'm not mistaken): https://cyclonedx.org/docs/1.4/json/